lib/sgn-sdk.ts

```diff
@@ -65,9 +65,10 @@ if (isBrowser()) {
 
         if (apiKey) scriptConfig.apiKey = apiKey;
 
-        if (trackId) scriptConfig.eventTracker = new Tracker({trackId});
+        if (trackId && !isStaging)
+            scriptConfig.eventTracker = new Tracker({trackId});
 
-        if (environment === 'staging') scriptConfig.coreUrl = coreUrlStaging;
+        if (isStaging) scriptConfig.coreUrl = coreUrlStaging;
```

Comment:
I would move this above new Tracker and after if apiKey. Weird to set it after new Tracker IMO